index.html

```diff
@@ -1949,56 +1949,573 @@ <h2>
+            DOMString organization;
+            DOMString recipient;
+            DOMString phone;
+          };
+        </pre>
+        <p>
+          A <a>AddressInit</a> is passed when <a data-lt=
+          "PaymentAddress.PaymentAddress()">constructing</a> a
+          <a>PaymentAddress</a>. Its members are as follows.
+        </p>
+        <dl data-dfn-for="AddressInit" data-link-for="" data-sort="ascending">
+          <dt>
+            <dfn>country</dfn> member
+          </dt>
+          <dd>
+            An <a>country</a>, represented as a a [[!ISO3166-1]] country code.
+          </dd>
+          <dt>
+            <dfn>addressLine</dfn> member
+          </dt>
+          <dd>
+            An <a>address line</a>, represented as sequence.
+          </dd>
+          <dt>
+            <dfn>region</dfn> member
+          </dt>
+          <dd>
+            A <a>region</a>.
+          </dd>
+          <dt>
+            <dfn>city</dfn> member
+          </dt>
+          <dd>
+            A <a>city</a>.
+          </dd>
+          <dt>
+            <dfn>dependentLocality</dfn> member
+          </dt>
+          <dd>
+            A <a>dependent locality</a>.
+          </dd>
+          <dt>
+            <dfn>postalCode</dfn> member
+          </dt>
+          <dd>
+            A <a>postal code</a>.
+          </dd>
+          <dt>
+            <dfn>sortingCode</dfn> member
+          </dt>
+          <dd>
+            A <a>sorting code</a>.
+          </dd>
+          <dt>
+            <dfn>languageCode</dfn> member
+          </dt>
+          <dd>
+            A <a>language code</a>, represented as a <a data-cite=
+            "!BCP47#section-2">language tag</a>.
+          </dd>
+          <dt>
+            <dfn>organization</dfn> member
+          </dt>
+          <dd>
+            An <a>organization</a>.
+          </dd>
+          <dt>
+            <dfn>recipient</dfn> member
+          </dt>
+          <dd>
+            A <a>recipient</a>. Under certain circumstances, this member may
+            contain multiline information. For example, it might contain "care
+            of" information.
+          </dd>
+          <dt>
+            <dfn>phone</dfn> member
+          </dt>
+          <dd>
+            A <a>phone number</a>, optionally structured to adhere to
+            [[!E.164]].
+          </dd>
+        </dl>
+      </section>
+      <section>
+        <h2>
+          Creating a <code>PaymentAddress</code> from user-provided input
+        </h2>
+        <p>
+          The steps to <dfn>create a <code>PaymentAddress</code> from
+          user-provided input</dfn> are given by the following algorithm. The
+          algorithm takes a <a>list</a> <var>redactList</var>, for which user
+          input will not be gathered.
+        </p>
+        <ol data-link-for="AddressInit">
+          <li>Let <var>details</var> be an object with no members that will
+          serve as a <a>AddressInit</a> dictionary.
```

Comment:
Just say "be an AddressInit dictionary with no members present". It's technically not an object since we're not in JS-land, we're in Web IDL-land.